src/applications/gi/components/profile/EstimatedBenefits.jsx

```diff
@@ -152,7 +152,7 @@ export const EstimatedBenefits = ({ profile, outputs, calculator }) => (
         visible={outputs.totalPaidToYou.visible}
       />
     </div>
-    <hr />
+    <hr aria-hidden="true" role="presentation" />
```

Comment:
I ran this through axe locally and it didn't like the `role="presentation"`. Please remove the role, and keep the `aria-hidden` and I'll fast approve.

Reply:
changed